### ISSUE TITLE
After %blockrun <button>, notebook jumps to bottom

### ISSUE THREAD

**Issue by rlele5** [kafonek/ipython_blocking#10]
Hi, 

Functionality-wise, I've been looking for this exact functionality for a while, so thanks for making this so easy with one line of code!!

One thing I've noticed though is that after the %blockrun cell executes, it immediately jumps to the end of the notebook. However the reason I use this is to wait for ipywidgets input. This means the user has to scroll all the way back up to the input cells to actually enter input. Do you know if there is a way to force the web page to "stop" at the %blockrun cell so that users can enter the input right then and there?

Thanks :)

**Comment by kafonek** [comment]
thanks for the kind words @rlele5 .  I'm not very familiar with how the front-end javascript of Notebook classic works, I think linking this issue and bringing up the topic in something like https://gitter.im/jupyter/jupyter or stackoverflow is your best hope on getting a definitive answer.  My general understanding is that the front-end javascript automatically scrolls to each cell as it's executed.  

From the front-end's perspective, `%blockrun` doesn't stop cell execution.  The code in each cell is indeed sent over websocket to the jupyter server just like code execution without `ipython_blocking` involved.  It's just that the actual execution of that code is deferred until the blocking action (widget change or whatever) is complete.  Then all the deferred code is executed and the output of each cell is filled out.

In summary, it might be possible to have the UI jump back to the cell where `%blockrun` or the widget target of blockrun is defined.  I just don't know how to do it.
